### ISSUE TITLE
Add EOL warning and disable Source Interface on instances running Ubuntu 16.04 after April 30

### ISSUE THREAD

**Issue by eloquence** [freedomofpress/securedrop#5688]
As we did with Trusty (#4325), for security reasons, we'll want to disable the use of the Source Interface on instances that are running on Ubuntu 16.04 after its end-of-life (April 30, 2021). Support for Ubuntu 20.04 is expected to ship with SecureDrop 1.8.0 (tentatively scheduled for 2021-02-23). 

See #4305 and #4325 for previous discussions regarding the language for the message displayed on the Source Interface. If there is no reason to modify this language, we may be able to re-use translations.

**Comment by eloquence** [comment]
(If we stick to a 6-8 week cadence, there's a good chance there'll be another release before April 30, so not strictly a release blocker for 1.8.0.)

**Comment by eloquence** [comment]
@emkll and I discussed this a bit today. All things considered, we agreed it would be ideal to update the warning messaging for 1.8.0, to ensure that all admins are reminded about the Ubuntu 20.04 upgrade as soon as it's possible to migrate. This also avoids boxing us in for the 1.9.0 release schedule.

Functionally, this could be done as follows:
1. Updating the current v2/v3 messaging (i.e. what was implemented in #5679) and pointing it to the [combined EOL advisory](https://securedrop.org/news/securedrop-advisory-end-life-v2-onion-services-and-ubuntu-1604/)
2. Updating the advisory with links to additional documentation as we publish it
3. Including similar code to #4325 to disable the Source Interface on Ubuntu 16.04 instances after April 30.

Open Q: How should the messaging differ, depending on instance state?
- v2-only, Ubuntu 16.04
- v2/v3, Ubuntu 16.04
- v3-only, Ubuntu 16.04

(No messages should be displayed on Ubuntu 20.04, which is v3-only.)

**Comment by eloquence** [comment]
@creviera has committed to work on this. For now we've agreed that for the 1.8.0 release, we can
- Remove all banners related to v2/v3 (if folks haven't done it yet by March, they can do it as part of the Ubuntu migration)
- Display a single banner that informs people about the April 30 end-of-life
- Implement the logic to disable the Source Interface after April 30

Essentially, we will want to replicate the functionality previously implemented in #4325 and #4055. Perhaps we can have a quick check-in if we want to stick to previous messaging/styling choices.

**Comment by eloquence** [comment]
For the "Learn More" link, I've set up https://securedrop.org/xenial-eol, which redirects to our current advisory blog post. My suggestion would be to keep that advisory up-to-date as we add more instructions, instead of creating a new one.

**Comment by sssoleileraaa** [comment]
So we're okay with this language on a disabled Source Interface:

```
We're sorry, our SecureDrop is currently offline.

Please try again later. Check our website for more information.
```

With the benefit of reusing previous translations?

**Comment by emkll** [comment]
The existing language seems clear to me, barring any objections from you @creviera .

As for the translations, it seems likely that these strings were never actually translated: the original change was merged to develop and introduced in 0.12.2 (https://github.com/freedomofpress/securedrop/commits/release/0.12.2) and it seems like no translations were done for 0.12.2, and the disabling logic was very swiftly removed from the `develop` branch: less than 2 weeks later in https://github.com/freedomofpress/securedrop/pull/4416/files
